src/components/common/PostalCode/postal-code.scss

```diff
@@ -0,0 +1,52 @@
+@import "src/styles/scaffold";
+
+.postal-code-input {
+  // Lighthouse
+  margin-top: 1rem;
+
+  [data-store-label] {
+    font-family: Arial; /* stylelint-disable-line */ // Prevent layout shift
+  }
+
+  [data-store-input] {
+    border: var(--fs-border-width-default) solid var(--fs-border-color-default);
+    border-radius: var(--fs-border-radius-default);
+
+    // Lighthouse
+    height: var(--fs-control-tap-size); /* stylelint-disable-line */
+    margin-left: 1rem;
+    font-family: Arial; /* stylelint-disable-line */ // prevent layout shift
+  }
+}
+
+[data-postal-code-bar] {
```

Comment:
after push the latest changes from master (commit [a17f249](https://github.com/vtex-sites/base.store/pull/424/commits/a17f249960b4345d32cf6497a3fc66aecfa29a50), this data attribute isn't working anymore. I couldn't find out why yet.

Reply:
I think I know what happened, on line 25 we're using `the `.button` selector but that class has been removed from the `Button` component. I think it should work again if you remove the `.button` from `.button[data-store-button][data-fs-button-variant="tertiary"]`.